rust/ql/consistency-queries/CfgConsistency.ql

```diff
@@ -26,8 +26,13 @@ query predicate scopeNoFirst(CfgScope scope) {
   not scope = any(ClosureExpr c | not exists(c.getBody()))
 }
 
+/** Holds if  `be` is the `else` branch of a `let` statement that results in a panic. */
+private predicate letElsePanic(BlockExpr be) {
+  be = any(LetStmt let).getLetElse().getBlockExpr() and
+  exists(Completion c | CfgImpl::last(be, _, c) | completionIsNormal(c))
```

Comment:
This looks fine for now, we don't really have a way to identify "panic"-like statements. A real panic wouldn't complete normally, it would abort the program. When we extract types, we may refine this to hold if the type of the expression is `!`.